contracts/attesters/hydra-s1/HydraS1SimpleAttester.sol

```diff
@@ -51,7 +51,7 @@ contract HydraS1SimpleAttester is IHydraS1SimpleAttester, HydraS1Base, Attester
   uint256 public immutable AUTHORIZED_COLLECTION_ID_FIRST;
   uint256 public immutable AUTHORIZED_COLLECTION_ID_LAST;
 
-  mapping(uint256 => address) internal _ticketsDestinations;
+  mapping(uint256 => address) internal _nullifiersDestinations;
```

Comment:
You should add some layout empty spaces here. Otherwise we won't be able to add new mapping in the HydraS1SimpleAttester in the future